code/modules/effects/map_effects/window_spawner.dm

```diff
@@ -175,6 +175,28 @@
 	icon_state = "full_i_rwindow"
 	window_path = /obj/structure/window/full/reinforced/indestructible
 
+//Coalition window frames
+/obj/effect/map_effect/window_spawner/full/reinforced/coalition
```

Comment:
these should all be a subtype of `/obj/effect/map_effect/window_spawner/full/shuttle`
icon states and grille/firedoor/etc subtypes are fine to keep for ease of use
`frame_path` should be normal window frames
`frame_color` set to `COLOR_COALITION`
`window_path` should be either normal reinforced or borosilicate, but I would prefer normal